docs/Reference/EthSigner-CLI.md

```diff
@@ -219,3 +221,65 @@ Timeout in milliseconds for requests to the Hashicorp Vault server. Default is 1
 ```bash tab="Example"
 --timeout=5000
 ```
+
+## Azure Options 
+
+### client-id
```

Comment:
Should this talk about how these values are determined from the Azure Portal?

Reply:
Yes - I've got a second ticket to add more information about actually using it.  Thought I'd get this one in first though.